__tests__/app.test.js

```diff
@@ -45,24 +41,44 @@ describe("get/api", () => {
   });
 });
 
-describe("get/articles/id", () => {
-  test("should give a status 200", () => {
-    return request(app).get("/api/articles/1").expect(200);
-  });
-  test("Should return correct article when given an id.", () => {
+describe("get/api/articles/:id", () => {
+  test("GET:200 Should return correct article when given an id.", () => {
     return request(app)
       .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
         expect(body[0].article_id).toBe(1);
+        expect(body[0].hasOwnProperty);
       });
   });
   test("GET:400 sends an appropriate status and error message when given an invalid id", () => {
     return request(app)
-      .get("/api/articles/9999")
+      .get("/api/articles/not-an-id")
       .expect(400)
       .then((response) => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
 });
+
+describe("get/api/articles", () => {
+  test("GET:200 Should an array of object containing article data", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body[0]).toMatchObject({
```

Comment:
because were getting all the articles, you should probably do a forEach and make sure all the articles match what you would expect

Reply:
Look at jest matches such as toMatchObject and expect.any